src/Tizen.Multimedia.Remoting/WebRTC/WebRTCDataChannel.cs

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using Tizen.Applications;
+using static Interop;
+
+namespace Tizen.Multimedia.Remoting
+{
+    /// <summary>
+    /// Provides the ability to control WebRTC data channel.
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public partial class WebRTCDataChannel : IDisposable
+    {
+        private readonly IntPtr _handle;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WebRTCDataChannel"/> class.
+        /// </summary>
+        /// <remarks>
+        /// The bundle is similar format as the RTCDataChannelInit members outlined https://www.w3.org/TR/webrtc/#dom-rtcdatachannelinit.<br/>
+        /// The following attributes can be set to options by using <see cref="Bundle"/> API:<br/>
+        /// 'ordered' of type bool            : Whether the channel will send data with guaranteed ordering. The default value is true.<br/>
+        /// 'max-packet-lifetime' of type int : The time in milliseconds to attempt transmitting unacknowledged data. -1 for unset. The default value is -1.<br/>
+        /// 'max-retransmits' of type int     : The number of times data will be attempted to be transmitted without acknowledgement before dropping. The default value is -1.<br/>
+        /// 'protocol' of type string         : The subprotocol used by this channel. The default value is NULL.<br/>
+        /// 'id' of type int                  : Override the default identifier selection of this channel. The default value is -1.<br/>
+        /// 'priority' of type int            : The priority to use for this channel(1:very low, 2:low, 3:medium, 4:high). The default value is 2.<br/>
+        /// </remarks>
+        /// <param name="webRtc">The owner of this WebRTCDataChannel.</param>
+        /// <param name="bundle">The data channel option. This can be null.</param>
+        /// <param name="label">The name of this data channel.</param>
+        /// <exception cref="ArgumentNullException">The webRtc or label is null.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public WebRTCDataChannel(WebRTC webRtc, Bundle bundle, string label)
```

Comment:
I think the label parameter(mandatory) should be before the bundle parameter which is optional.
It also meet with web API and native API.
: https://www.w3.org/TR/2013/WD-webrtc-20130910/#widl-RTCPeerConnection-createDataChannel-RTCDataChannel-DOMString-label-RTCDataChannelInit-dataChannelDict

Reply:
OK I'll fix it.